### PR TITLE
(IAC-144) - Removal of Inappropriate Terminology

### PR DESCRIPTION
**Pull request by david22swan**
Removal of any inappropriate or offensive terminology from the code base

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/master?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/v7w84n21w9yjspq7/branch/main?svg=true)](https://ci.appveyor.com/project/puppetlabs/pdk-planning/branch/main)
```
Comment (pmcmaw) [inline README.md:1]: This URL doesn't seem to exist, it can't find build.

**Review by pmcmaw — CHANGES_REQUESTED** [review]
Minor change required where build cannot be found ('Build not found or access denied.')

Other than that it looks great.

File: README.md (hunk 1 above)
Comment (david22swan) [reply README.md:1]: That's strange, it should have been created by the prs appveyor run

Comment (david22swan) [reply README.md:1]: Hmmm, from what I can tell it should be created when the PR is merged in and the main becomes the current build

Comment (pmcmaw) [reply README.md:1]: Cool well were about to find out. Merging.

**Review by pmcmaw — APPROVED** [review]

**Comment by pmcmaw** [comment]
You were correct the URL is fine now it has been merged, good shout @david22swan 👍

**Pull request merged**